internal/usecases/parkingspace/get_single_parking_space_test.go

```diff
@@ -0,0 +1,63 @@
+package usecases_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/IgorSteps/easypark/internal/domain/entities"
+	usecases "github.com/IgorSteps/easypark/internal/usecases/parkingspace"
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockParkingSpaceRepository struct {
```

Comment:
I am not sure where did you get this from, but why make your own mocks if they are generated for you?